src/AElf.WebApp.Application.Chain/BlockChainAppService.cs

```diff
@@ -665,5 +667,17 @@ private async Task<MethodDescriptor> GetContractMethodDescriptorAsync(Address co
 
             return null;
         }
+        
+        private void ValidateMessage(IMessage message,int errorCode)
```

Comment:
return true/false is better for "ValidateMessage", the user can decide whether to throw an exception or logging error/warn